scripts/patch-realsense-ubuntu-lts-hwe.sh

```diff
@@ -76,6 +76,10 @@ if [[ ( ${xhci_patch} -eq 1 ) && ( ${k_maj_min} -ne 404 ) ]]; then
 	echo -e "\e[43mThe xhci_patch flag is compatible with LTS branch 4.4 only, currently selected kernel is $(uname -r)\e[0m"
 	exit 1
 fi
+k_tick=$(echo ${kernel_version[2]} | awk -F'-' '{print $2}')
+# For version 5.15.0-72 hid patch already applied
+[ $k_maj_min -gt 515 ] && skip_hid_patch=1
```

Comment:
How about 5.19? we probably have versions that do not include the patch, right?

Is it the same for focal / jammy?

Do we also need to change the other (none HWE) script?

Reply:
This will apply for 5.19 also.
I have 5.19 prepared but we need to merge this first.